glossary.yml

```diff
@@ -5560,27 +5651,28 @@
   en:
     term: "vignette"
     def: >
-      A long-form guide used to provide details of a package beyond the README.md or
+      A long-form guide used to provide details of a [package](#package) beyond the README.md or
       function documentation.
 
 
 - slug: vim
   en:
     term: "Vim (editor)"
     def: >
-      The default text editor on Unix. "How do I exit the Vim editor?" is one of the
-      most popular questions on [Stack Overflow](#stack_overflow).
+      The default text editor on Unix. Vim is a very powerful text editor that allows
+      the user to execute shell commands and use [regular expressions](#regular_expression) to
+      alter files programmatically. 
```

Comment:
I say this as a user of Vim for the past three years: I think it is definitely worth keeping some sense of how difficult it is for newcomers to use Vim.

Reply:
That is fine. I also use Vim. But what was there before:

```
"How do I exit the Vim editor?" is one of the most popular questions on
[Stack Overflow](#stack_overflow).
```

is unnecessarily negative. I, for one, have never been stuck in Vim, but for AGES, could only exit nano by closing the terminal window.

I don't want the gist of the entry to be like a lot of responses to [this twitter thread](https://twitter.com/cybercdh/status/1318887202680721409) where someone asked about cool Vim shortcuts, and like 2/3 of the replies are just ways to exit Vim (and open nano).

What about:

```
Vim is a very powerful text editor with a steeper learning curve than nano,
but that allows the user to execute shell commands and use
[regular expressions](#regular_expressions) to alter files programmatically.
```

Reply:
Oof... that thread is not fun and I agree that the original text was passive-aggressive.

I think your suggested text manages expectations well.

Reply:
I was so disappointed by that thread, because I was so excited to maybe learn something new.